src/Web/Event/CustomEvent.purs

```diff
@@ -12,3 +13,15 @@ fromEvent = unsafeReadProtoTagged "CustomEvent"
 
 toEvent :: CustomEvent -> Event
 toEvent = U.unsafeCoerce
+
+foreign import new :: String -> CustomEvent
+
+-- | Create a new `CustomEvent`, storing some data in its `detail` field,
+-- | and using defaults for everything else.
+new' :: forall a. String -> Nullable a -> CustomEvent
```

Comment:
We generally don't expose `Nullable` directly when we can expose `Maybe` instead. Can we expose `Maybe` interfaces for things exported from this module? (Nullable is fine for FFI internals)